### DISCUSSION TITLE
With Celery only the initial image is being created

### DISCUSSION THREAD

**Discussion by 1r00t** [codingjoe/django-pictures#77] [Q&A]
I added Celery to my project and when I upload a picture, only the original is being uploaded.

```
kroko@wedding:~/wedding-spa$ ll /var/www/media/pictures/
total 12256
drwxr-xr-x 2 root root    4096 Nov 18 18:56 ./
drwxr-xr-x 3 root root    4096 Nov 18 18:53 ../
-rw-r--r-- 1 root root 6248960 Nov 18 18:53 P1110218.JPG
-rw-r--r-- 1 root root 6290432 Nov 18 18:56 P1110225.JPG
```

No aspect ratio directories get created.

Maybe I'm doing something wrong, I'm new to Celery.

**Comment by 1r00t** [comment]
When I use Dramatiq it is working fine

**Comment by codingjoe** [comment]
Hi @1r00t,

Thank you for reaching out about this problem. Hm… this behavior is covered by tests, and should work. Are you certian that the tasks are being executed? Would you mind sharing your Celery setup, or try creating another task, see if it runs? Please also make sure your workers are listening to the queue `pictures`.

Cheers,
Joe

**Comment by codingjoe** [comment]
@1r00t is this still open? I presume my reply helped debugging the issue, so I'll close this for now, OK?

**Comment by 1r00t** [comment]
I stuck with dramatiq so I can't really say. Sorry for not helping. Time is scarce atm
